examples/random-dog/dog-image-widget/src/widget/widget.css

```
@@ -0,0 +1,23 @@
+body {
+  margin: 0px;
+}
+
+.main-wrapper {
+    overflow: hidden;
+    position: relative;
+}
+
+.new-dog-btn {
+    background-color: rgba(100, 44, 165);
+    border: none;
+    color: white;
+    left: 20px;
+    position: absolute;
+    top: 20px;
+}
+
+.dog-img {
+    object-fit: cover;
+}
```

Update css so image covers entire widget area

```suggestion
.main-wrapper {
    height: 100%;
    left: 0px;
    position: absolute;
    top: 0px;
    width: 100%;
}

.image-wrapper {
    height: 100%;
    width: 100%;
}

.new-dog-btn {
    background-color: rgba(100, 44, 165);
    border: none;
    color: white;
    left: 20px;
    position: absolute;
    top: 20px;
}

.dog-img {
    height: 100%;
    object-fit: cover;
    width: 100%;
}
```